src/Symfony/Contracts/Service/Attribute/Autoconfigure.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Contracts\Service\Attribute;
```

Comment:
I've added the `Autoconfigure` attribute because I believe it conveys semantics that could benefit any DI container. But I'm fine moving it to the DI component if that's preferred.

Reply:
(but this would prevent using the attribute without the DI component - while e.g. the Console component would benefit from the semantics to define commands' name + description + etc)